index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>YouTube Clone</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/d7c2fd4a1e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <header>
+        <nav id="grid-container-header">
+            <div class="logo-container">
+                <span class="black"><i class="fa fa-bars" aria-hidden="true"></i></span>
+                <i class="fa fa-youtube-play fa-lg" aria-hidden="true"><span class="black">YouTube Clone</span></i>
+            </div>
```

Comment:
- [ ] This needed to be an image

![image](https://user-images.githubusercontent.com/61479648/90419338-d0b83d80-e08c-11ea-879b-d2460888fdf5.png)